File: Java/백준/Bronze/27959. 초코바/초코바.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int N = sc.nextInt();
        int M = sc.nextInt();
        
        int result = (N * 100) - M;
        
        if (result >= 0) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
        
        sc.close();
    }
}

File: C++17/백준/Bronze/27959. 초코바/초코바.cc
#include <iostream>
using namespace std;

int main (void) {
    int N, M;
    cin >> N >> M;
    
    int result = (N * 100) - M;
    
    if (result >= 0) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }
    
    return 0;
}